src/sentry/models/groupsubscription.py

```diff
@@ -117,10 +121,28 @@ def bulk_subscribe(
                     is_active=True,
                     reason=reason,
                 )
-                for user_id in user_ids
-                if user_id not in existing_subscriptions
+                for user_id in user_ids.difference(existing_subscriptions)
             ]
 
+            existing_team_subscriptions = set(
```

Comment:
i think i changed my mind and we should add in a feature flag for this code inside bulk_subscribe. i don't anticipate it causing any problems, but i think we should err on the side of caution

Reply:
fair enough, will make the change in a sec